tests/DMSDocumentTest.php

```diff
@@ -38,88 +38,11 @@ public function testGetRelatedDocumentsIsExtensible()
     {
         DMSDocument::add_extension('StubRelatedDocumentExtension');
 
-        $emptyDocument = new DMSDocument;
-        $relatedDocuments = $emptyDocument->getRelatedDocuments();
+        $document = DMSDocument::create();
+        $relatedDocuments = $document->getRelatedDocuments();
 
         $this->assertCount(1, $relatedDocuments);
-        $this->assertSame('Extended', $relatedDocuments->first()->Filename);
-    }
-
-    /**
-     * Ensure that the DMS Document CMS actions contains a grid field for managing related documents
-     */
-    public function testDocumentHasCmsFieldForManagingRelatedDocuments()
-    {
-        $document = $this->objFromFixture(DMSDocument::class, 'document_with_relations');
-        $gridField = $this->getRelatedDocumentsGridField($document);
-        $this->assertInstanceOf(GridField::class, $gridField);
-
-        $gridFieldConfig = $gridField->getConfig();
-
-        $this->assertNotNull(
-            GridFieldAddExistingAutocompleter::class,
-            $addExisting = $gridFieldConfig->getComponentByType(GridFieldAddExistingAutocompleter::class),
-            'Related documents GridField has an "add existing" autocompleter'
-        );
-
-        $this->assertNull(
-            $gridFieldConfig->getComponentByType(GridFieldAddNewButton::class),
-            'Related documents GridField does not have an "add new" button'
-        );
-    }
-
-    /**
-     * Ensures that the DMS Document CMS Related and Versions fields are removed if user can't edit
-     */
-    public function testDocumentHasNoCMSFieldsForManagingRelatedDocumentsIfCantEdit()
```

Comment:
These tests are worth keeping as they're testing the permission model. We're verifying that the custom permissions work (which is likely an important feature)